### PR TITLE
If the element is not IMG or IFRAME, just DIV

### PR DESCRIPTION
**Pull request by RodyRafa**
Hi @verlok can i do some suggestion for your lazy-load project?

When i using just div to know if is in the screen and won't put any attribute =D

### CODE REVIEW

File: src/lazyload.setSources.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ export const setSources = function (element, settings) {
     const {data_src: dataSrc, data_srcset: dataSrcSet} = settings;
     const tagName = element.tagName;
     const elementSrc = getData(element, dataSrc);
+    if(!src) {
```
Comment (verlok) [inline src/lazyload.setSources.js]: I can't understand how can `(!src)` be ever true, here, since that variable is not declared anywhere.

**Comment by verlok** [comment]
Also, I didn't understand what you're trying to achieve here. Maybe if you ask me, I have the solution already.

**Comment by RodyRafa** [comment]
🤦‍♂️ Sorry, I forgot!

**Comment by verlok** [comment]
I think you were rrying to achieve what I then released in version 10.4.1 and 8.6.0. 
The LazyLoad can now be used to just know when a simple `div` entered the viewport, in order to LazyLoad pretty much everything, aka the Disqus comment section in a blog page.
I’m gonna close the PR then. If you remember what you were trying to achieve and it’s not what I already did, feel free to reopen.

**Pull request closed without merging**